<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rear_ADAS_setting</class>
 <widget class="QDialog" name="rear_ADAS_setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="titleButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>121</width>
         <height>51</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>121</width>
         <height>51</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(71, 154, 255);
color:rgb(255,255,255)</string>
       </property>
       <property name="text">
        <string>后ADAS设置</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>498</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>303</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" rowspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>371</width>
       <height>351</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>371</width>
       <height>351</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox{
border-width:2px;
border-style:solid;
border-radius: 10px;
border-color:blue;
margin-top:0.5ex;
}
font: 75 10pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" rowstretch="8,1" columnstretch="1,9,1">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="highLabel">
          <property name="styleSheet">
           <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;color:rgb(255,255,255)</string>
          </property>
          <property name="text">
           <string>高</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QSlider" name="high_low_Slider">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lowLabel">
          <property name="styleSheet">
           <string notr="true">color:rgb(255,255,255)</string>
          </property>
          <property name="text">
           <string>低</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QWidget" name="widget" native="true">
        <widget class="Line" name="HorizontalLine">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>260</y>
           <width>301</width>
           <height>10</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color:
rgb(255, 8, 12)</string>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="Line" name="VerticelLine">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>10</width>
           <height>241</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(255, 2, 18)</string>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>33</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="leftLabel">
          <property name="styleSheet">
           <string notr="true">color:rgb(255,255,255)</string>
          </property>
          <property name="text">
           <string>左</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="left_right_Slider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="rightLabel">
          <property name="styleSheet">
           <string notr="true">color:rgb(255,255,255)</string>
          </property>
          <property name="text">
           <string>右</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>28</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
     <zorder>widget</zorder>
     <zorder>horizontalSpacer</zorder>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>303</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="returnButton">
     <property name="minimumSize">
      <size>
       <width>41</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>41</width>
       <height>41</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/icon/Back.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="nextButton">
     <property name="minimumSize">
      <size>
       <width>81</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>81</width>
       <height>41</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-style: none;
	border: 0px;
	color: #F0F0F0;
	padding: 5px;	
	border-radius:5px;
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #0F7DBE, stop:1 #1582C3); 
}

.QPushButton[focusPolicy=&quot;0&quot;] {
	border-style: none;
	border: 0px;
	color: #F0F0F0;
	padding: 0px;	
	border-radius:3px;
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #0F7DBE, stop:1 #1582C3); 
}

.QPushButton:hover{ 
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #50A3F0, stop:1 #489CEA);
}

.QPushButton:pressed{ 
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #0F7DBE, stop:1 #1582C3);
}</string>
     </property>
     <property name="text">
      <string>下一步</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
